client/src/main/java/com/aws/greengrass/cli/commands/DeploymentCommand.java

```diff
@@ -125,6 +127,7 @@ public DeploymentCommand(NucleusAdapterIpc nucleusAdapterIpc) {
         Optional<Path> artifactDirPath = deTilde(artifactDir);
         createLocalDeploymentRequest.setArtifactsDirectoryPath(artifactDirPath.isPresent() ?
                 artifactDirPath.get().toString() : null);
+        createLocalDeploymentRequest.setFailureHandlingPolicy(failureHandlingPolicy == null ? FailureHandlingPolicy.ROLLBACK : failureHandlingPolicy);
```

Comment:
setting the default to rollback is wrong. The default should be the _current_ behavior, which is to not rollback.